app/main.py

```diff
@@ -2,0 +1,40 @@
+class Animal:
+    def __init__(
+        self, name: str, appetite: int, is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> None:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
```

Comment:
The `feed` method is defined to return `None`, but it actually returns an integer value. Consider changing the return type annotation to `int`.